Sources/Operation.swift

```diff
@@ -25,6 +25,19 @@ public struct Operation {
     /// Declares this operation to be deprecated. Usage of the declared operation should be refrained. 
     /// Default value is false.
     public let deprecated: Bool
+
+    /// A unique string used to identify the operation
+    public let operationId: String?
+
+    /// A list of tags used to group operations together
+    public let tags: [String]?
+
+    /// A list of which security schemes are applied to this operation.
+    /// The list of values describes alternative security schemes that can be used 
+    /// (that is, there is a logical OR between the security requirements).
+    /// This definition overrides any declared top-level security.
+    /// To remove a top-level security declaration, an empty array is used.
+    public let security: [SecurityRequirement]?
```

Comment:
Ah, I didn't realize the nil-vs-empty array had semantic meaning here. That's fine. I don't think it does for the tags though.